From 8d4add222e0e3463c0f866f3b0d8da2022a1dac9 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Fri, 24 Aug 2018 22:48:08 +0000
Subject: [PATCH] TOOLS-2070 tools/mk/Makefile.targ should conditionally set
 BUILD Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com> Approved
 by: Jordan Hendricks <jordan.hendricks@joyent.com>

---
 tools/mk/Makefile.targ | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tools/mk/Makefile.targ b/tools/mk/Makefile.targ
index dac448a..8ae89c2 100644
--- a/tools/mk/Makefile.targ
+++ b/tools/mk/Makefile.targ
@@ -73,6 +73,9 @@
 #
 #	BASH		Path to bash (default: "bash")
 #
+#	BUILD		top-level directory for node binaries, generated docs,
+#			and any other build output (default: "build")
+#
 #	CSCOPE_DIRS	Directories to search for source files for the cscope
 #			index. (default: ".")
 #
@@ -126,7 +129,7 @@ endif
 #
 # Defaults for other fixed values.
 #
-BUILD		= build
+BUILD		?= build
 DISTCLEAN_FILES += $(BUILD)
 DOC_BUILD	= $(BUILD)/docs/public
 
-- 
2.21.0

